executor/show.go

```diff
@@ -760,10 +728,7 @@ func (e *ShowExec) getTable() (table.Table, error) {
 }
 
 func (e *ShowExec) appendRow(row []interface{}) {
-	if !e.forChunk {
-		e.rows = append(e.rows, types.MakeDatums(row...))
-		return
-	}
+	e.rows = append(e.rows, types.MakeDatums(row...))
```

Comment:
`e.rows` only used by `Next` and can be removed since `Next` is removed.

Reply:
Anything else? I'll see.